app/BUILD.bazel

```diff
@@ -262,7 +262,11 @@ for file in VIEW_MODELS_WITH_RESOURCE_IMPORTS]
 
 # TODO(#1617): Remove genrules post-gradle
 '''
+<<<<<<< HEAD
```

Comment:
Please remove merge artifacts (& in general make sure to for future changes).

Reply:
Ackkk I always miss the merge artifacts in comments - I will be more careful!